WildAidDemo/functions/getAgencies/config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "getAgencies",
+    "private": false,
+    "run_as_system": true
```

Comment:
Does it need to `run_as_system`? That means that it won't apply the Realm rules to ensure that people only see the documents that they're entitled to. If someone edited the web source code then a user would be allowed to see data that they're not entitled. If a particular aggregation needs to run as system, then it could be done within a `private` function that's invoked from a public function (without `run_as_system` set). Initially, I'd try setting `run_as_system` to `false` and if that doesn't work, move components of this function to private functions. Alternatively, add checks in here and filter out any data for agencies that this user isn't entitled to access

Reply:
That is need to working $Search functionality and aggregation pipeline. In another case it is not working and returns error.